soql-standalone-parser/src/main/scala/com/socrata/soql/parsing/AbstractParser.scala

```diff
@@ -200,9 +200,16 @@ abstract class AbstractParser(parameters: AbstractParser.Parameters = AbstractPa
       }
     }
 
-  def namedSelection = expr ~ opt(AS() ~> simpleUserIdentifier) ^^ {
+  private val standardSystemColumns = Set(":id", ":created_at", ":updated_at")
```

Comment:
Not a fan of this; if you're going to do it this way, have the set of standard system columns passed in as a parameter when the parser is constructed.

Reply:
Yeah, that will be cleaner.  I will update it.